src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimator.h

```diff
@@ -49,7 +49,7 @@ namespace OpenMS
   /**
     @brief This class represents the abstract base class of a signal to noise estimator.
 
-    A signal to noise estimator should provide the signal to noise ratio of all raw data points
+    A signal to noise estimator should provide the signal to noise ratio of all profile data points
```

Comment:
check if this needs to be profile data

Reply:
@hroest - Is profile here OK?

Reply:
I think its ok but we also use it for chromatograms which are neither in centroid or profile mode ...

Reply:
Then raw data seems to fit better in the case of SignalToNoiseEstimator - imho.